wouso/interface/cpanel/views.py

```diff
@@ -1046,6 +1047,22 @@ def get_success_url(self):
     GroupsAdd.as_view()
 )
 
+@permission_required('config.change_setting')
+def race_toggle_can_play(request):
+    try:
+        race_pk = int(request.GET.get('id', None))
+    except:
+        raise Http404
+
+    race = get_object_or_404(Race, pk=race_pk)
+
+    race.can_play = not race.can_play
+    race.save()
+
+    data = {}
+    data['can_play'] = race.can_play
+
+    return HttpResponse(json.dumps(data))
```

Comment:
@catalin-oancea, if the data response is simple, you can just use json.dumps({'can_play': race.can_play}). No need for the data variable.